Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -497,6 +536,29 @@ theorem _root_.Matrix.posDef_inv_iff [DecidableEq n] {M : Matrix n n 𝕜} :
     letI := (Matrix.isUnit_nonsing_inv_iff.1 <| h.isUnit).invertible
     Matrix.inv_inv_of_invertible M ▸ h.inv, (·.inv)⟩
 
+lemma posDef_sqrt [DecidableEq n] {M : Matrix n n 𝕜} (hM : M.PosDef) :
+    PosDef hM.posSemidef.sqrt := by
+  unfold PosSemidef.sqrt
+  apply PosDef.mul_mul_conjTranspose_same
```

Comment:
```suggestion
  apply PosDef.mul_mul_conjTranspose_same
```

Reply:
I think I was getting frustrated at unification here, and didn't clean up afterwards!